tests/ModelContextProtocol.Tests/Server/McpServerToolTests.cs

```diff
@@ -355,6 +355,27 @@ public async Task CanReturnCallToolResponse()
         Assert.Equal("image", result.Content[1].Type);
     }
 
+    [Fact]
+    public async Task SupportsSchemaCreateOptions()
+    {
+        AIJsonSchemaCreateOptions schemaCreateOptions = new ()
+        {
+            RequireAllProperties = false
```

Comment:
We're in the process of changing this default to `false` in MEAI, so once that gets incorporated this test would stop testing what it is meant to be testing. Would it be possible to replace this with something else? I recommend using a custom `TransformSchemaNode` that simply inserts a bogus keyword into the schema node that is then asserted for.